IdentityCore/src/webview/embeddedWebview/MSIDOAuth2WebviewController.m

```diff
@@ -29,9 +29,18 @@
 
 @implementation MSIDOAuth2WebviewController
```

Comment:
nit: can we call it embedded controller to be clear?

Reply:
sure